data_rentgen/server/services/get_user.py

```diff
@@ -20,13 +20,15 @@ async def wrapper(
         auth_provider: Annotated[AuthProvider, Depends(Stub(AuthProvider))],
         access_token: Annotated[str | None, Depends(oauth_schema)],
     ) -> User:
+        # keycloak provider patches session and store access_token in cookie,
         # dummy auth stores access_token in "Authorization" header
+        access_token = request.session.get("access_token", "") or access_token
         user = await auth_provider.get_current_user(
             access_token=access_token,
             request=request,
         )
         if user is None:
-            raise EntityNotFoundError("User not found")  # type: ignore[call-arg]
+            raise EntityNotFoundError("User", "username", "None")  # type: ignore[call-arg]
```

Comment:
Shouldn't this exception be moved to auth_provider? Exception needs to get some identifying information for user, e.g. user id or username